internal/workflow/preprocessing.go

```diff
@@ -72,28 +61,21 @@ func (w *PreprocessingWorkflow) Execute(ctx temporalsdk_workflow.Context, params
 	// Remove hidden files.
 	var removedPaths activities.RemovePathsResult
 	e = temporalsdk_workflow.ExecuteActivity(withLocalActOpts(ctx), remove.RemoveFilesName, &remove.RemoveFilesParams{
-		RemovePath: extractPackageRes.Path,
+		RemovePath: localPath,
 		IgnorePath: removePath,
 	}).Get(ctx, &removedPaths)
 	if e != nil {
 		return nil, e
 	}
 
-	// Repackage MOMA Sip into a Bag.
-	var sipCreation activities.SipCreationResult
-	e = temporalsdk_workflow.ExecuteActivity(withLocalActOpts(ctx), activities.SipCreationName, &activities.SipCreationParams{
-		SipPath: extractPackageRes.Path,
-	}).Get(ctx, &sipCreation)
-	if e != nil {
-		return nil, e
-	}
+	// TODO: repackage MOMA Sip into a Bag.
 
-	realPath, e := filepath.Rel(w.sharedPath, sipCreation.NewSipPath)
+	relPath, e := filepath.Rel(w.sharedPath, localPath)
 	if e != nil {
 		return nil, e
 	}
 
-	return &PreprocessingWorkflowResult{RelativePath: realPath}, e
+	return &PreprocessingWorkflowResult{RelativePath: relPath}, e
```

Comment:
Currently, we are no modifying the initial relative path, so we could just return `params.RelativePath`.